models/snapchat.yml

```diff
@@ -117,15 +117,15 @@ models:
       - name: currency
         description: The current used by the account in Snapchat.
       - name: spend
-        description: The spend on the ad in the given day.
+        description: The spend on the campaign in the given day.
       - name: impressions
-        description: The number of impressions the ad had on the given day.
+        description: The number of impressions the campaign had on the given day.
       - name: swipes
-        description: The number of swipes the ad had on the given day.
+        description: The number of swipes the campaign had on the given day.
       - name: conversion_purchases
-        description: The total number of purchases for an ad on the given day.
+        description: The total number of purchases for the campaign on the given day.
       - name: conversion_purchases_value
-        description: The total value of purchases for an ad on the given day.
+        description: The total value of purchases for campaign campaign on the given day.
```

Comment:
Could you note the docs update in the CHANGELOG?

Reply:
updated and took care of the typo here